docs/specs/toc/toc.yml

```diff
@@ -1522,3 +1522,16 @@ outputs:
         "a.md": [{ "source": "TOC.yml", "type": "metadata" }]
       }
     }
+---
```

Comment:
Also add a test case with `output-type=pagejson`

Reply:
I will change the default output-type as "pageJson" and then no need to add new spec tests for it.